examples/2serious_mnist.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+import os
+import sys
+sys.path.append(os.getcwd())
+sys.path.append(os.path.join(os.getcwd(), 'test'))
+
+import numpy as np
+from tinygrad.tensor import Tensor, GPU
+from tinygrad.nn import BatchNorm2D
+from tinygrad.utils import get_parameters
+from test_mnist import fetch_mnist, train, evaluate
+import tinygrad.optim as optim
+GPU = os.getenv("GPU", None) is not None
+QUICK = os.getenv("QUICK", None) is not None
+
+class SqueezeExciteBlock2D:
+  def __init__(self, filters):
+    self.filters = filters
+    self.weight1 = Tensor.uniform(self.filters, self.filters//32)
+    self.bias1 = Tensor.uniform(1,self.filters//32)
+    self.weight2 = Tensor.uniform(self.filters//32, self.filters)
+    self.bias2 = Tensor.uniform(1, self.filters)
+
+  def __call__(self, input):
+    se = input.avg_pool2d(kernel_size=(input.shape[2], input.shape[3])) #GlobalAveragePool2D
+    se = se.reshape(shape=(-1, self.filters))
+    se = se.dot(self.weight1) + self.bias1
+    se = se.relu() 
+    se = se.dot(self.weight2) + self.bias2
+    se = se.sigmoid().reshape(shape=(-1,self.filters,1,1)) #for broadcasting 
+    se = input.mul(se)
+    return se
+
+class ConvBlock:
+  def __init__(self, h, w, inp, filters=128, conv=3):
+    self.h = h
+    self.w = w
+    self.filters = filters
+    self.conv = conv
+    self.inp = inp
+    self.c1 = Tensor.uniform(filters, inp, conv, conv)
```

Comment:
Biases on the convs?

Reply:
I definitely missed that ones.... keras hides so many things! That's why this is a good exercise!